src/styles/container.scss

```
@@ -0,0 +1,9 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 50px auto 0;
+  padding: 0 120px;
+  box-sizing: content-box;
+  max-width: 1024px;
```

```suggestion
  max-width: 1024px;
```
don't hardcode max-width ([checklist 13](https://github.com/mate-academy/layout_catalog-test_task/blob/solution-to-review/checklist.md))